### PR TITLE
[TW-1050] Removed mentions of team activity on Homepage

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
Users can no longer view their team's activity from the Homepage

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/navigating-postman.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The header enables you to create workspaces, access reports, explore the public
 <img alt="Postman header left side" src="https://assets.postman.com/postman-docs/postman-desktop-app-header-v10.10.jpg" width="350px"/>
 
 * **&#8592; &#8594;** - Navigate backward and forward through pages you've visited within Postman. _(Postman desktop app only)_
-* **Home** - Go to your personal home page, which includes alerts, announcements, your activity feed, your recently visited workspaces, and links to resources for [your team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) if applicable.
+* **Home** - Go to your personal home page, which includes alerts, announcements, your recently visited workspaces, and links to resources for [your team](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/) if applicable.
```
Comment (sourabhbagrecha) [inline src/pages/docs/getting-started/navigating-postman.md:62]: LGTM 👍🏼

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 2]
```diff
@@ -79,10 +79,6 @@ To filter by element, select **Elements** at the top of the activity feed and se
 
 > You can access the changelog of a specific collection by selecting **View Changelog** next to actions on collections.
 
-## Viewing team activity
```
Comment (sourabhbagrecha) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:82]: LGTM 👍🏼

**Review by sourabhbagrecha — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
I added a few changes - LGTM

**Pull request merged**